smf/sled-agent/gimlet/config.toml

```diff
@@ -0,0 +1,19 @@
+# Sled Agent Configuration
+
+# Identifies the revision of the sidecar that is attached, if one is attached.
+# TODO: This field should be removed once Gimlets have the ability to auto-detect
+# this information.
+sidecar_revision = "rev_a"
```

Comment:
Do we have any `a`s left?  Maybe this should default to `b`?

Reply:
@Nieuwejaar - do you have an opinion on this?

Reply:
We only have a single rev A left in use on the bench and it is slated for removal next week according to @arjenroodselaar. I would not propagate Rev A as otherwise it means we have to make mods to work in every other env.

Reply:
I'll pull this into https://github.com/oxidecomputer/omicron/pull/2534 to be reviewed separately, but updating this SGTM.

Reply:
Seems fine to me.